yb_migrate/cmd/exportData.go

```diff
@@ -66,8 +67,10 @@ func exportData() {
 		err := exec.Command("touch", exportDir+"/metainfo/flags/exportDataDone").Run() //to inform import data command
 		utils.CheckError(err, "", "couldn't touch file exportDataDone in metainfo/flags folder", true)
 		color.Green("Export of data complete \u2705")
+		log.Infof("Export of data completed.")
```

Comment:
Use `log.Info()` when you have to output only a string. `Infof()` will hard-working function that assumes first argument to be a format string and it sequentially scans it in search of format specifiers (e.g. %s, %v, etc.)